app/fonts.styl

```diff
@@ -0,0 +1,64 @@
+@font-face {
```

Comment:
why here we use  curly braces `{}` and in other files we don't ? I would try to stick with one convention